### PR TITLE
Usefull tips when merging upstream changes into our version of HB.

### PR DESCRIPTION
**Pull request by milo-hyben**

### CODE REVIEW

File: hail.md [hunk 1]
```diff
@@ -385,6 +385,82 @@ git merge upstream/main  # Potentially resolve any conflicts.
...
+
+   ```bash
+   [merge]
+      conflictstyle = diff3
+   [diff]
+      colorMoved = true
+      indentHeuristic = true
+      tool = difftastic
+      wordRegex = \\w+|\\W
+      #??? wsErrorHighlight = new
```
Comment (jmarshall) [inline hail.md:400]: Only really important line here is `merge.conflictstyle = diff3` (or maybe even suggest `zdiff3`).

`diff.colorMoved` is generally useful but not really too important for doing these merges. And the rest are specific to my environment or the default anyway.

Comment (milo-hyben) [reply hail.md]: Thanks John! I had updated the file.

File: hail.md [hunk 2]
```diff
@@ -414,6 +491,40 @@ This will print a link to the [CI dashboard](https://ci.hail.populationgenomics.
...
+Sometimes there is a need to view SQL content of Hail Batch database. To be able to do that, add the following function to your local .zshrc file:
+
+```bash
+kube-hail() {
+# Find all pods that match the input name, get the first one
+local pod_name=$(kubectl get pods --no-headers=true | grep "$1" | head -n 1 | awk '{print $1}')
+
+# Check if a pod name was found
+if [ -z "$pod_name" ]; then
+echo "No matching pod found"
```
Comment (jmarshall) [inline hail.md:505]: Looks like some indentation fell off here (and off the three lines under `else`), but no biggie.

**Review by jmarshall — APPROVED** [review]
Good stuff, and all accurate. Thanks!

File: hail.md (hunk 2 above)
Comment (milo-hyben) [reply hail.md]: It would work as is, it is bash code, but good point. Had updated it.

**Pull request merged**